package.json

```diff
@@ -99,7 +99,7 @@
     "@libp2p/tcp": "^10.0.7",
     "@libp2p/websockets": "^9.0.6",
     "@multiformats/multiaddr": "^12.3.1",
-    "@oceanprotocol/lib": "^3.4.6",
+    "@oceanprotocol/lib": "^4.0.2",
```

Comment:
Can you check this PR https://github.com/oceanprotocol/vscode-extension/pull/79, it seems that it has a typo in vscode's tsconfig for `CommonJS`. If it seems fine, maybe we can approve that one and merge it.